hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -448,6 +451,45 @@ public void incNumSnapshotListFails() {
     numSnapshotListFails.incr();
   }
 
+  public void setNumSnapshotActive(long num) {
+    long currVal = numSnapshotActive.value();
+    numSnapshotActive.incr(num - currVal);
```

Comment:
Serviceable. But I wonder if there is a better alternative to `MutableCounterLong` in this case.

Reply:
@smengcl In the initial commit, I declared the new metrics as [gauges](https://github.com/apache/ozone/blob/b961f153b6c0244f5d30802be118cfe780393db7/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java#L124) instead of counters. I changed them to counters because it seems cleaner in cases such as creating a snapshot where we need to increment `NumSnapshotActive` or in deleting a snapshot where we need to decreament `NumSnapshotActive` and increment `NumSnapshotDeleted`.

Reply:
Got it. Sounds good.